csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs

```diff
@@ -131,6 +132,40 @@ public DependencyManager(string srcDir, IDependencyOptions options, ILogger logg
                 progressMonitor.UnresolvedReference(r.Key, r.Value);
             }
 
+            var webViewExtractionOption = Environment.GetEnvironmentVariable("CODEQL_EXTRACTOR_CSHARP_STANDALONE_EXTRACT_WEB_VIEWS");
```

Comment:
I think in the long run we'll have a couple of configuration options, so it definitely makes sense to group them. At the same time, we should try to minimize the number of options as the extractor should "just work". I don't have full confidence in this cshtml to C# generation, so I wanted to hide it behind an undocumented feature flag, but eventually I don't think there's any reason to not have this run all the time.

Reply:
That makes sense!
In any case, we could consider to do the environment variable read in the options class instead (this might make it easier to test the entire DependencyManager class at some point in time), but we don't have to do it now.